contracts/libraries/math/MathOperation.sol

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: BSD-3-Clause
 pragma solidity 0.8.17;
 
-library Math {
+library MathOperation {
```

Comment:
Why changing name?

Reply:
Because there is an issue when we updated `OpenZeppelin` library to the newest one. There is also a `Math` library there and build fails because of that. With `Math` name, build fails with message:
```
Error: 
Compiler run failed
error[2333]: DeclarationError: Identifier already declared.
 --> test/liquidityMining/MiningCalculation.t.sol:6:1:
  |
6 | import "contracts/libraries/math/MiningCalculation.sol";
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Note: The previous declaration is here:
 --> node_modules/@openzeppelin/contracts/utils/math/Math.sol:9:1:
  |
9 | library Math {
  | ^ (Relevant source part starts here and spans across multiple lines).
```
That's why I changed the name from `Math` to `MathOperation` to solve this issue.